src/providers/AlertProvider.tsx

```diff
@@ -42,273 +56,256 @@ type AlertProviderProps = {
-                {(alert.actions ?? []).map(({ title, onPress, icon, primary, danger, backgroundColor }) => (
-                  <Pressable
-                    key={title}
-                    onPress={() => {
-                      onPress?.();
-                      hideAlert();
-                    }}
-                    style={({ pressed }) => [
-                      styles.button,
-                      primary && styles.primaryButton,
-                      primary && {
-                        backgroundColor: backgroundColor ? backgroundColor : colors.primary,
-                      },
-                      danger && {
-                        backgroundColor: "#b62000",
-                      },
-                      {
-                        opacity: primary ? (pressed ? 0.6 : 1) : (pressed ? 0.3 : 0.6),
-                      }
-                    ]}
-                  >
-                    {icon ? icon : null}
-
-                    <Text style={[styles.buttonText, { color: colors.text }, primary && styles.primaryButtonText]}>
-                      {title}
-                    </Text>
-                  </Pressable>
-                ))}
+              <View
+                style={[
+                  styles.buttons,
+                  {
+                    borderColor: colors.border,
+                    flexDirection:
+                      (alert.actions ?? []).length > 2 ? "column" : "row",
+                    alignItems: "center",
+                  },
+                ]}
+              >
+                {alert.actions?.map(
+                  ({
+                    title,
+                    onPress,
+                    icon,
+                    primary,
+                    danger,
+                    backgroundColor,
+                  }) => (
+                    <Pressable
+                      key={title}
+                      onPress={() => {
+                        hideAlert();
+                        onPress?.();
+                      }}
+                      disabled={delays[title] > 0}
+                      style={({ pressed }) => [
+                        styles.button,
+                        {
+                          width:
+                            (alert.actions ?? []).length > 2 ? "100%" : "auto",
+                          justifyContent: "center",
+                          alignItems: "center",
+                          opacity: pressed
+                            ? 0.6
+                            : delays[title] === 0
+                              ? 1
+                              : 0.5,
+                        },
+                        primary
+                          ? {
+                            backgroundColor:
+                                backgroundColor ?? colors.primary,
+                          }
+                          : danger
+                            ? { backgroundColor: "#FC1E0D" }
```

Comment:
```suggestion
                            ? { backgroundColor: "#BE0B00" }
```

Cette couleur serait mieux, elle est moins flash et c'est celle utilisé dans Papillon.